lisa/node.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         self.capability = capability
         self.name: str = ""
         self.index = index
+        self.is_ready: bool = False
```

Comment:
Change it's name from `is_ready` to `is_connected` to make it more accurate and same as new environment status.

Reply:
And reset it to False in `close`.